src/compute/src/logging/compute.rs

```diff
@@ -131,14 +131,21 @@ pub struct Peek {
     id: GlobalId,
     /// The logical timestamp requested.
     time: Timestamp,
+    /// The type of the peek: currently 'index' or 'persist'.
+    peek_type: &'static str,
```

Comment:
Can you check that this doesn't increase the size of `ComputeEvent`? If it doesn't change the size, this should be OK, but if it increases the size I'd prefer to have an enum here with `repr(u8)` and see if it fits. `ComputeEvent` is somewhat performance sensitive, and we'd want to be careful not increasing its size too much.

Reply:
Good to know! Added a test for `ComputeEvent` size; this change increases it, but so does just replacing it with a `repr(u8)`. Hoisting the type field up to the `ComputeEvent` variant itself does _not_ increase, the size, though - pushed a commit with that change. Overall not any more complicated I think!